frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -86,67 +97,97 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
     handleReset();
   }, [handleReset]);
 
+  const formEditor = (
```

Comment:
Definitely need to be creating files for this... no reason to have this file get any bigger. Look to make a Component that takes the props it needs to craft its output.

Reply:
Moved to a new file.